src/main/java/com/blackducksoftware/integration/jira/issue/handler/JiraIssueHandler.java

```diff
@@ -278,15 +277,15 @@ private void addComment(final BlackDuckIssueModel blackDuckIssueModel, final Str
         logger.debug(String.format("Attempting to add comment to %s: %s", issueKey, comment));
         if (StringUtils.isNotBlank(comment) && !checkIfAlreadyProcessedAndUpdateLastBatch(blackDuckIssueModel)) {
             final String newCommentKey = String.valueOf(comment.hashCode());
-            final String storedLastCommentKey = issueServiceWrapper.getIssueProperty(issue.getId(), BlackDuckJiraConstants.BLACKDUCK_JIRA_ISSUE_LAST_COMMENT_KEY);
+            final String storedLastCommentKey = issueServiceWrapper.getIssuePropertyWrapper().getIssueProperty(issue.getId(), issueServiceWrapper.getJiraAdminUser(), BlackDuckJiraConstants.BLACKDUCK_JIRA_ISSUE_LAST_COMMENT_KEY);
```

Comment:
We should get the user from `JiraUserContext` so that we don't necessarily have to depend on the service wrapper for this.

Reply:
Otherwise, if we have it in the service, we shouldn't require it as a param.